RNASeqTools_TSS/rnaseqtools_tss.jl

```diff
@@ -0,0 +1,105 @@
+# # RNASeq data analysis using RNASeqTools
+#
+# Requirements:
+#
+# * Julia 1.8.5
+# * RNASeqTools 0.3.4
+# * sra-tools 3.0.3
+# * samtools 1.6
+# * bwa-mem2 2.2.1
+#
+# In this tutorial, we will first download a few files (genome, annotation, sequencing reads) with sra-tools. 
+# We will then align the reads to the genome using bwa-mem2. We will then use the various Packages from BioJulia 
+# through the interface of RNASeqTools to load the alignments file, compute the coverage from it and do some 
+# simple signal detection in the coverage.
+
+using Pkg
+Pkg.add(url="https://github.com/maltesie/RNASeqTools#v0.3.4")
+
+using RNASeqTools
+
+# ## Downloading the data
+#
+# Our data will be from V. Cholerae, we need a genome and its annotation: First go to [NCBI](https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_013085075.1/),
+# after clicking Download, select .fasta and .gff and click Download again. Then extract the content of the archive 
+# directory /ncbi_dataset/data/GCF_013085075.1/ into the folder containing this notebook. The folder should then
+# look like:
```

Comment:
Not critical, but we can do all of these things using julia's `Cmd` functionality. I can take a stab at this a bit later.

Reply:
Cool, thanks. While playing with BioServices.jl, I found a way to download annotations and genomes with the EUtils module.  Sadly, it does not support gff3, though :(